03-attribute-operations.qmd

```diff
@@ -126,14 +137,14 @@ world[['name_long', 'geometry']]
 world.loc[:, 'name_long':'pop']
 ```
 
-Subsetting by a boolean series:
+Subsetting by a list of boolean values (0 and 1 or True and False):
```

Comment:
The intention was that the index is typically not being typed (as in a `list`) but created through conditions (in which case it's going to be a `Series`), so the `Series` example would be more "realistic". Does that make sense?

Reply:
Fair enough, I agree that it's an unnecessary complication of the code, please keep the `list` version, thanks!